java/test/apps/PanelPro/PanelProTest.java

```diff
@@ -1,28 +1,125 @@
 package apps.PanelPro;
 
 import java.awt.GraphicsEnvironment;
-import jmri.util.JUnitUtil;
+import java.io.*;
+
+import org.apache.commons.io.*;
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Assume;
 import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Test;
 
+import jmri.util.JUnitUtil;
+import jmri.util.JmriJFrame;
+import jmri.util.JUnitAppender;
+import jmri.util.JUnitUtil;
+
 /**
- *
+ * This is more of an acceptance test than a unit test. It confirms that the entire
+ * application can start up and configure itself.
+ * 
  * @author Paul Bender Copyright (C) 2017
+ * @author Bob Jacobsen Copyright (C) 2017
  */
 public class PanelProTest {
 
-    @Test
-    @Ignore("Causes Exception")
     public void testCTor() {
         Assume.assumeFalse(GraphicsEnvironment.isHeadless());
         PanelPro t = new PanelPro();
         Assert.assertNotNull("exists", t);
     }
 
+    @Test
+    public void testLaunch() throws IOException {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+                
+        try {
+            // create a custom profile
+            FileUtils.copyDirectory(new File("java/test/apps/PanelPro/profiles/LocoNet_Simulator"), new File("temp/LocoNet_Simulator"));
```

Comment:
JUnit provides a rule-based temporary folder mechanism that automatically cleans itself after the test. You might want to look at jmri.jmrit.roster.RosterTest for an example of using it. See also http://junit.org/junit4/javadoc/latest/org/junit/rules/TemporaryFolder.html